static/bidder-info/mobfoxpb.yaml

```diff
@@ -1,6 +1,6 @@
 endpoint: "http://bes.mobfox.com/?c=__route__&m=__method__&key=__key__"
 maintainer:
-  email: "platform@mobfox.com"
+  email: "support@mobfox.com"
```

Comment:
sent email. Please reply with received.

Reply:
@bretg Requesting you to please reply to the email address.

Reply:
It's not my response you want. I pinged the mobfox team.

Reply:
gotcha. I generally ask the author of the PR for reply. Since in this case, it was you so tagged you.